web/hook.go

```diff
@@ -10,6 +10,8 @@ import (
 
 	log "github.com/Sirupsen/logrus"
 	"github.com/gin-gonic/gin"
+	"github.com/hashicorp/go-version"
+	"fmt"
 )
```

Comment:
convention to group stdlib packages at the top of the import, then external packages separated by newline, like this:

```
import (
    "fmt"
    "net"
    "net/http"

    "github.com/gin-gonic/gin"
    "github.com/hashicorp/go-version"
)
```